packages/core/helper-plugin/src/utils/tests/stopPropagation.test.ts

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+import { screen, render } from '@testing-library/react';
+
+import { stopPropagation, onRowClick, StopPropagation } from '../stopPropagation';
+
+describe('HELPER-PLUGIN | utils | stopPropagation', () => {
```

Comment:
```suggestion
describe('stopPropagation', () => {
```

Jest does a lot of the naming stuff for us